benchmarks/asv.conf.json

```diff
@@ -0,0 +1,98 @@
+    // (if remote), or by looking for special directories, such as
+    // ".git" (if local).
+    "dvcs": "git",
+
+    // The tool to use to create environments.  May be "conda",
+    // "virtualenv" or other value depending on the plugins in use.
+    // If missing or the empty string, the tool will be automatically
+    // determined by looking for tools on the PATH environment
+    // variable.
+    "environment_type": "virtualenv",
+
+    // the base URL to show a commit for the project.
+    "show_commit_url": "https://github.com/nipreps/nifreeze/commit/",
+
+    // The Pythons you'd like to test against.  If not provided, defaults
+    // to the current version of Python used to run `asv`.
+    // "pythons": ["3.12"],
+
+    // The matrix of dependencies to test.  Each key is the name of a
+    // package (in PyPI) and the values are version numbers.  An empty
+    // list indicates to just test against the default (latest)
+    // version.
+    "matrix": {
+        "dipy": [],
+        "nipype": [],
+        "nest-asyncio": [],
+        "nitransforms": [],
+        "numpy": [],
+        "scikit_learn": [],
+        "scipy": []
+    },
+
+    // The directory (relative to the current directory) that benchmarks are
+    // stored in.  If not provided, defaults to "benchmarks"
+    "benchmark_dir": "benchmarks",
+
+    // The directory (relative to the current directory) to cache the Python
+    // environments in.  If not provided, defaults to "env"
+    "env_dir": "env",
+
+
+    // The directory (relative to the current directory) that raw benchmark
+    // results are stored in.  If not provided, defaults to "results".
+    "results_dir": "results",
+
+    // The directory (relative to the current directory) that the html tree
+    // should be written to.  If not provided, defaults to "html".
+    "html_dir": "html",
+
+    // The number of characters to retain in the commit hashes.
+    // "hash_length": 8,
+
+    // `asv` will cache wheels of the recent builds in each
+    // environment, making them faster to install next time.  This is
+    // number of builds to keep, per environment.
+    "build_cache_size": 8,
+
+    // The commits after which the regression search in `asv publish`
+    // should start looking for regressions. Dictionary whose keys are
+    // regexps matching to benchmark names, and values corresponding to
+    // the commit (exclusive) after which to start looking for
+    // regressions.  The default is to start from the first commit
+    // with results. If the commit is `null`, regression detection is
+    // skipped for the matching benchmark.
+    //
+    // "regressions_first_commits": {
+    //    "some_benchmark": "352cdf",  // Consider regressions only after this commit
+    //    "another_benchmark": null,   // Skip regression detection altogether
+    // }
+
+    // Maximum time in seconds that a benchmark is allowed to run before it is terminated.
+    "default_benchmark_timeout": 240,
```

Comment:
```suggestion
    "default_benchmark_timeout": 240
```

Reply:
Sorry, when creating an initial, empty `asv` config file, the tool adds commas after every value, including the last item. You can check this locally. So I am only following the pattern across the rest of the file here.

Reply:
I don't know what they use for parsing the JSON, but any standard parser will crash with this one. Assuming their parser does not crash with valid JSON, I'd go with valid JSON. (In addition, this error trips highlighters such as GitHub's)

Reply:
Other major tools like `NumPy` or `scikit-learn` seem to keep the commas, but OK, I'd rather use an automatic tool to lint the JSON file then. Can you please propose one?

Reply:
I don't think we use sufficient JSONs as to set up a linter, it's just that if we catch that ASV's config initialization is lazy we should not inherit a malformed JSON object. I honestly doubt we are going to keep editing this file.

A plausible scenario is that the start using a proper parser in some future version of avs and then our tests break because this JSON was malformed all the way through. It's just about keeping things tidy.

Reply:
OK, removed. Force pushed.

Reply:
It looks like they are planning to move to toml. I just realized that the comments are also disallowed in JSON (this could be read with something like hJSON). Let's move on and how they adopt toml soon.